sites/docs/pages/reference/themes/index.md

```diff
@@ -0,0 +1,526 @@
+---
+sidebar_position: 4
+hide_table_of_contents: false
+title: Themes
+description: Customize the appearance of your Evidence application in light and dark mode using our theming system.
+sidebar_badge: New!
+---
+
+Customize the appearance of your Evidence application in light and dark mode using our theming system.
+
+# Appearance
+
+Evidence supports three appearance modes: `light`, `dark`, and `system`. An unconfigured Evidence application will use `light` mode and not allow switching appearances. Evidence includes a built-in dark mode without any configuration necessary.
```

Comment:
I had purposefully phrased it this way ("An unconfigured Evidence application") because the template does include themes/dark mode by default. So the phrase "By default" depends on when your app was created for it to be true.